web/app/scripts/views/record/add-edit-partial.html

```diff
@@ -42,7 +42,7 @@
                             </div>
                         </div>
                         <div class="col-md-12">
-                            <div class="map" leaflet-map driver-embed-map style="height:300px;"></div>
+                            <div class="map" leaflet-map driver-embed-map editable="true"></div>
```

Comment:
Yeah, that definitely makes sense for your use case. Mine is a bit different since the details tabs are being rendered dynamically based on record content and the map data is already available at that time. I think it's fine as-is though, we can add that ability later on if we want.